roles/servicetelemetry/templates/manifest_snmp_traps.j2

```diff
@@ -0,0 +1,29 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: '{{ meta.name }}-snmp-webhook'
+  namespace: '{{ meta.namespace }}'
+spec:
+  selector:
+    matchLabels:
+      app: '{{ meta.name }}-snmp-webhook'
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: '{{ meta.name }}-snmp-webhook'
+    spec:
+      containers:
+        - name: 'prometheus-webhook-snmp'
+          image: 'quay.io/infrawatch/prometheus-webhook-snmp:latest'
```

Comment:
We're going to have to be pretty careful here since we'll need to make sure this is updated somehow for downstream releases. Not exactly sure yet how this should work, but at the very least we need to account for this in the downstream bundle manifests / CSV as well and add our downstream links to the `relatedImages` list for the Service Telemetry Operator.

Likely need to track this in our release documentation which I created yesterday. This is mostly a note-to-self to make sure we account for this before merging to master and doing an STF release.

Reply:
I guess this would/should work like with all other containers we are pulling for downstream.